exchange-client.gemspec

```diff
@@ -0,0 +1,30 @@
+# coding: utf-8
+lib = File.expand_path('../lib', __FILE__)
+$LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
+require 'openstax/exchange/client/version'
+
+Gem::Specification.new do |spec|
+  spec.name          = "openstax-exchange-client"
```

Comment:
Regarding the actual Client module, I do feel it is unnecessary to put the entire gem in it.